livre/core/util/Utilities.cpp

```diff
@@ -39,6 +40,49 @@ bool hasOpenGL42Support()
 #endif
 }
 
+Vector2ui computeFrameRange( const Vector2ui& frameRange,
+                             const Vector2ui& boundaries )
+{
+    // frame range is outside the boundaries
+    if( frameRange[1] <= boundaries[0] || frameRange[0] >= boundaries[1] )
+        return INVALID_FRAME_RANGE;
+
+    if( frameRange[0] < boundaries[0] || frameRange[1] > boundaries[1] )
+        LBVERB << "Clamping the requested frame range to the boundaries"
```

Comment:
LBERROR? Not sure. But in any case print the boundaries or the final range that was computed.